packages/colorpickers/src/styled/ColorDialog/StyledDialogPreview.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { IRGBColor } from '../../elements/ColorPicker/reducer';
```

Comment:
Would it be useful to have a single `types.ts` file for these shared definitions? These values are used in many areas/props besides the reducer.

Reply:
A new `types.ts` has been created for shared definitions. I've removed the `exports` that aren't used and aren't intended for consumer usage.